synapse/cortex.py

```diff
@@ -2451,6 +2451,10 @@ async def _initCoreViews(self):
             ldef = {'name': 'default'}
             ldef = await self.addLayer(ldef=ldef, nexs=False)
             layriden = ldef.get('iden')
+
+            role = await self.auth.getRoleByName('all')
+            await role.addRule((True, ('layer', 'read')), gateiden=layriden, nexs=False)
```

Comment:
Do we need to retroactively add this permission as well to the ``all`` rule for existing cortexes?

Reply:
I debated how to do this, we can discuss